src/Language/Visitor/EnterReturnValue.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Digia\GraphQL\Language\Visitor;
+
+final class EnterReturnValue extends AbstractReturnValue
+{
+
+    public const ACTION_SKIP = 'SKIP';
```

Comment:
What is this used for?

Reply:
They're not used anywhere, I just copied the values from the reference implementation